command_line/indexed_as_integrated.py

```diff
@@ -0,0 +1,73 @@
+import iotbx.phil
+
+import dials.util
+from dials.array_family import flex
+from dials.util.options import OptionParser, reflections_and_experiments_from_files
+
+help_message = """
+Small modifications to an indexed spot list to allow it to be treated as if
+it were integrated, for rapid feedback analysis.
+"""
```

Comment:
Can we add a warning (to both the help and the program stdout) that this is very much experimental and not (yet) intended for routine use, etc etc?

Reply:
Wilco

Reply:
You happy with the health warnings just added?

Since I had to figure out the logging I added a few extra things as well:

```
The following parameters have been modified:

input {
  experiments = indexed.expt
  reflections = indexed.refl
}

DIALS (2018) Acta Cryst. D74, 85-97. https://doi.org/10.1107/S2059798317017235
DIALS 3.dev.633-g55edc3010

----------------------------------------------------------
Do not use the output for structure solution or refinement
----------------------------------------------------------

Saved 10883 of 11185 reflections


Saved pseudo-integrated data to: pseudo_integrated.refl


```